__tests__/server.test.js

```diff
@@ -111,3 +111,61 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("should return 200 on successful call", () => {
+    return request(app).get("/api/articles/1/comments").expect(200);
+  });
+
+  test("should return a list of all the comments attached to that article id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments.length).toBe(11);
+        body.comments.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id");
```

Comment:
Please add the following to your test: 

 - check that the typeof the property value is what you expect as well as checking that is has the property keys you expect.